crates/hardforks/src/hardfork/ethereum.rs

```diff
@@ -49,6 +49,17 @@ hardfork!(
         Prague,
         /// Osaka: <https://eips.ethereum.org/EIPS/eip-7607>
         Osaka,
+        // BPOs: <https://eips.ethereum.org/EIPS/eip-7892>
+        /// BPO 1
+        Bpo1,
+        /// BPO 2
+        Bpo2,
+        /// BPO 3
+        Bpo3,
+        /// BPO 4
+        Bpo4,
+        /// BPO 5
+        Bpo5,
```

Comment:
these aren't useful as dedicated variants because they just are timestamp values that influence active blobparams

adding them as fork enums will cause a lot of friction and imo we can solve this differently because we only need these for forkid, imo we should track them as separate entries in the ChainHardforks struct and add a new classification for them,

```
struct Bpo {
  name, timestamp
}
```

that we then can use when we compute the forkid